cli-plugins/apps/command-reference/clustersupplychain.hbs.md

```diff
@@ -0,0 +1,43 @@
+# Tanzu Apps Cluster Supply Chain
+
+## tanzu apps cluster supply chain get
+
```

Comment:
I'd switch the order of the commands in this page.

lead with `list`
then suggest that the user do a get on any of the supply chains listed in order to view more detailed information about the selectors and conditions that must be met for a workload to be selected by that supply chain.